src/main/resources/db/migration/V83__Add_recall_notes_table.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE note (
+                                id UUID PRIMARY KEY,
+                                recall_id UUID NOT NULL REFERENCES recall (id),
+                                subject TEXT NOT NULL,
+                                details TEXT NOT NULL,
+                                index INTEGER NOT NULL,
+                                document_id UUID REFERENCES document (id),
+                                created_by_user_id UUID NOT NULL,
```

Comment:
should reference users table

Reply:
Good one, thanks, on it 👍

Reply:
I think this has been missed also for `rescind_record.created_by_user_id` @philsegal ?  Per: https://mojdt.slack.com/archives/C0255MPKHHC/p1645177103323509?thread_ts=1645136931.777759&cid=C0255MPKHHC